### DISCUSSION TITLE
App Complementare e utenti "non Amministratore"

### DISCUSSION THREAD

**Discussion by vincenzocoletta** [indomus/forum#2953] [GENERALE - Discussioni generiche]
Ciao, tema semplice sul quale chiedo come al solito agli esperti dopo aver prima sbattuto la testa sul pc e poi cercato disperatamente soluzioni online: ho due configurazioni distinte di HA ed utilizzo Companion per switchàre dall'una all'altra all'occorrenza. Però ho anche definito degli utenti non amministratori che quindi sull'app non possono entrare in impostazioni per configurare (e quindi poi per switchàre su) la seconda app... Qualcuno ha già risolto o secondo voi infattibile?

**Comment by snakuzzo** [comment]
Sinceramente, mai provato...ma è davvero essenziale per queste persone usare la companion app ?
Potresti semplicemente creare degli shortcut...quello che vedi da web è uguale a quello che vedi da app

**Reply by vincenzocoletta** [reply]
Si, potrebbe essere una alternativa ma mi piace un pò meno, soprattutto perché saprebbe di arresa :D
Però mi fai venire in mente un ulteriore tentativo prima di mollare: alzo i diritti ad amministratore, poi configuro sull'app il secondo HA e poi rimodifico i diritti. Se funziona con lo swipe a tre dita per passare dall'una all'altra senza quindi andare in impostazioni allora è bingo! ;)
Ne approfitto per integrare ancora con un punto sull'app: ma anche a voi risulta che la configurazione della plancia di default diversa dalla Panoramica oppure la "Modifica l'ordine e nascondi gli elementi dalla barra laterale" da menù utente siano legati alla sessione in corso (si resettano ad ogni disconnessione dell'utente)??
